bin/templates/cordova/lib/log.js

```diff
@@ -21,32 +21,23 @@
 
 var path = require('path');
 var os = require('os');
-var Q = require('q');
```

Comment:
I believe this should remain in this PR and would be removed in : https://github.com/apache/cordova-android/pull/833

Reply:
Module imports are removed _iff_ the module is not used. Otherwise ESLint would complain.